requirements.txt

```diff
@@ -1,8 +1,12 @@
-    pytket == 1.4.1
+[comment]: # if version are updated, update the version in the build-manual file, too
```

Comment:
Hmm. Both `check-examples.yml` and `check-manual.yml` do `pip install -r requirements.txt` so this won't work. They will both have to be changed. Perhaps we need an `install-requirements` script?

Reply:
I have just seen that, too. I have just added the same installation process in the same way, having a script might be better... Will do that now.